resources/prosody-plugins/util.lib.lua

```diff
@@ -597,7 +597,7 @@ function table_shallow_copy(t)
 end
 
 local function table_find(tab, val)
-    if not tab then
+    if not tab or not val then
```

Comment:
Isn't this dangerous, in case you are looking for a boolean in a table? If you store false you get back nil. Which is falsey, so maybe it doesn't matter?